### PR TITLE
README update

### PR DESCRIPTION
**Pull request by u-hubar**
Improvements:
- Updated README
- General repo structure improvement.

This PR fixes:
- #5

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,45 +1,221 @@
...
+
+The OriginTrail Decentralized Network implements the DKG according to the OriginTrail protocol.
+
+It is:
+
+* **a permissionless network** - anyone can run OriginTrail nodes
+* **a multi-chain data exchange network** - connects to several blockchains (currently Ethereum and xDai with more integrations upcoming such as with Polkadot)
+* **designed for off-chain data exchange using standardized data models** (GS1 & W3C standards and recommendations)
+* **public open source software**
+* **infrastructure for knowledge marketplaces & tenders** - more info [here](https://www.youtube.com/watch?v=4uCxYGRh5fk)
```
Comment (NZT48) [inline README.md:138]: This video is quite old, I would suggest to remove it

File: README.md [hunk 2]
```diff
@@ -1,45 +1,221 @@
...
+
+More information is available on the OriginTrail [website](https://origintrail.io), [official documentation](https://docs.origintrail.io) and [blog](https://medium.com/origintrail).
```
Comment (NZT48) [inline README.md:140]: Maybe to remove this also, and leave it only at the end of doc

File: README.md [hunk 3]
```diff
@@ -1,45 +1,221 @@
...
+                <br><b>Metaverse</b>
+            </td>
+        </tr>
+    </table>
+</div>
 
 
+</details>
 
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
```
Comment (NZT48) [inline README.md:202]: I would put less this back to top buttons, or generally remove them, not sure how much people use them

File: README.md [hunk 4]
```diff
@@ -1,45 +1,221 @@
-# DKG.py
+<a name="readme-top"></a>
+
+> **Disclaimer: Beta Version**
```
Comment (NZT48) [inline README.md:3]: In my opinion this disclaimer should be smaller, no need for so much explanation also we don't plan to keep this forever

File: README.md [hunk 5]
```diff
@@ -1,45 +1,221 @@
...
 
+Key characteristics of knowledge graphs include:
 
-## Intro - What is a Decentralized Knowledge Graph (DKG)
+- Prioritization of data connections, treating them as "first-class citizens" within the realm of linked data.
+- Designed to accommodate data from diverse sources, supporting a variety of formats.
+- Their data models are flexible and easily extendable, ready to adapt to the evolving nature of data and its relationships.
 
+While knowledge graphs are typically deployed within the boundaries of a single organization to capture knowledge from various internal and external sources, we conceptualize them more broadly. We define the **Decentralized Knowledge Graph (DKG)** as a globally-shared, common infrastructure for data exchange, benefiting both organizations and individuals.
```
Comment (NZT48) [inline README.md:108]: I am not sure about this definition, when we google definition for DKG first thing is from our docs, and it sounds better `a global, open data structure comprised of interlinked knowledge assets.`

**Review by NZT48 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (u-hubar) [reply README.md:138]: This whole part is outdated, I haven't updated it yet

File: README.md (hunk 5 above)
Comment (u-hubar) [reply README.md:108]: Fixed

**Review by NZT48 — APPROVED** [review]

**Review by NZT48 — APPROVED** [review]

**Pull request merged**